concepts/05 UI Components/Toast/00 Getting Started with Toast/05 Display a Toast notification/05 Use the 'notify' Method.md

```diff
@@ -0,0 +1,269 @@
+Call the **notify** method to display a Toast. This method can accept two different sets of arguments. 
+
+A basic syntax is [notify(message, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_type_displayTime). You specify the [message](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#message), [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type), and [displayTime](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#displayTime).
+
+To specify additional Toast properties, call the [notify(options, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_type_displayTime) method and pass an object as the first argument. The example below uses this syntax.
+
+You can specify one of the four predefined types of notifications, depending on the message:
+
+- *'info'*   
+A blue toast with a message bubble icon.
+
+- *'warning'*   
+A yellow toast with an exclamation mark icon.
+
+- *'error'*   
+A red toast with an **X** icon.
+
+- *'success'*   
+A green toast with a check mark icon.
+
+If you call the method that allows additional options, you can set the [width](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#width), [height](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#height), and [position](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#position) options. The configuration of the **position** property in the example below reads as follows: "place **my** *bottom* side **at** the *bottom* side **of** the *"#container"*.
```

Comment:
After reading this article it seems to me that only `width`, `height` and `position` properties can be passed inside of first parameter object, but we can pass any Toast property

Reply:
Noted